templates/model/pos.go

```diff
@@ -48,8 +48,8 @@ func (c *ConfigPos) AnnotateErr(err error) error {
 		return nil
 	}
 
-	pos := "(position unknown)" // This can happen when field values are defaults, rather that coming from the config file
-	if *c != (ConfigPos{}) {
+	pos := "(position unavailable)" // This can happen when field values are defaults, rather that coming from the config file
+	if c != nil && *c != (ConfigPos{}) {
```

Comment:
What are the situations in which `c` can be nil? Are there other places where we're assuming it's not nil?

Reply:
Currently this is just a convenience for tests. If you think it's important, I can update the tests to make it always non-nil, at the cost of a little verbosity.

Reply:
Nah - it's fine